integration/cluster.go

```diff
@@ -929,3 +938,42 @@ type grpcAPI struct {
 	// Auth is the authentication API for the client's connection.
 	Auth pb.AuthClient
 }
+
+// copyTLSFiles clones certs files to dst directory.
```

Comment:
`util_test.go` or some other test file for these copy functions since they're not used by the integration package outside of tests?